Makefile

```
@@ -202,3 +202,7 @@ JSONNETFMT_CMD := $(JSONNETFMT) -n 2 --max-blank-lines 2 --string-style s --comm
 .PHONY: jsonnet-fmt
 jsonnet-fmt: | $(JSONNETFMT)
 	PATH=$$PATH:$(BIN_DIR):$(FIRST_GOPATH)/bin echo ${JSONNET_SRC} | xargs -n 1 -- $(JSONNETFMT_CMD) -i
+
+.PHONY: rules/rules.go
+rules/rules.go: $(OAPI_CODEGEN)
+	$(OAPI_CODEGEN) -generate types,client,chi-server -package rules -o rules/rules.go rules/spec.yaml
```

This should likely be "simplified" as such so that the output file is always in sync with the target name
```suggestion
	$(OAPI_CODEGEN) -generate types,client,chi-server -package rules -o $@ rules/spec.yaml
```